pkg/controllers/chart/chart.go

```diff
@@ -566,9 +580,6 @@ func args(chart *v1.HelmChart) []string {
 	if spec.TargetNamespace != "" {
 		args = append(args, "--namespace", spec.TargetNamespace)
 	}
-	if spec.Repo != "" {
-		args = append(args, "--repo", spec.Repo)
```

Comment:
see above - we are now prefixing the repo name instead of passing the URL as a flag.

I suspect that because we were always passing this flag, the repo definition that we added was never being used at all.